app/scripts/controller-init/messengers/keyring-controller-messenger.ts

```diff
@@ -17,7 +13,7 @@ export type KeyringControllerMessenger = ReturnType<
  * messenger.
  */
 export function getKeyringControllerMessenger(
-  messenger: Messenger<AllowedActions, AllowedEvents>,
+  messenger: Messenger<never, never>,
```

Comment:
It was using the wrong types from a different controller. `KeyringController` does not require any `AllowedActions` or `AllowedEvents`